src/mds/SimpleLock.h

```diff
@@ -167,7 +167,7 @@ class SimpleLock {
   __s16 state;
 
 private:
-  __s16 num_rdlock;
```

Comment:
__s32 should be enough

Reply:
Thanks:-) I think __s64 maybe more safe, after all, it wouldn't do anything bad. Right?

Reply:
class CInode include several locks, we want to keep this data structure as small as possible.